docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -122,10 +122,94 @@ Starting in version 8.4, all default ingest pipelines call a non-existent and no
 If left uncreated, this pipeline has no effect on your data. However, if added to a data stream and customized,
 this pipeline can be used for custom data processing, adding fields, sanitizing data, and more.
 
-The full name of the `@custom` pipeline follows the following pattern: `<type>-<dataset>@custom`.
-The `@custom` pipeline can directly contain processors or you can use the
-pipeline processor to call other pipelines that can be shared across multiple data streams or integrations.
-The `@custom` pipeline will persist across all version upgrades.
+Staring in version 8.12, ingest pipelines can be configured to process events at various levels of customization.
+
+`@global`::
```

Comment:
The name of the global custom pipeline is `global@custom`

Reply:
This is fixed by [d8bf6e1](https://github.com/elastic/ingest-docs/pull/675/commits/d8bf6e10217ffe3ccb54f265515d3b59fa61bd12)